mobile/src/components/courses-management/courses-management.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FC } from 'react';
+
+import { AppScreenName, DataStatus } from '~/common/enums/enums';
+import { CourseGetResponseDto } from '~/common/types/types';
+import {
+  CategoryCell,
+  Pagination,
+  ScrollView,
+  Spinner,
+  View,
+} from '~/components/common/common';
+import {
+  useAppDispatch,
+  useAppNavigate,
+  useAppSelector,
+  useCallback,
+  useFocusEffect,
+  usePagination,
+} from '~/hooks/hooks';
+import { coursesActions, coursesManagementActions } from '~/store/actions';
+
+import { DEFAULT_PAGE_SIZE } from './common/constants/constants';
+import { ActionCell } from './components/action-cell/action-cell';
+import { CoursesManagementTable } from './courses-management-table/courses-management-table';
+import { styles } from './styles';
+
+const CoursesManagement: FC = () => {
+  const { page, handlePageChange } = usePagination();
+  const navigation = useAppNavigate();
+  const dispatch = useAppDispatch();
+  const { dataStatus, coursesWithCategory, totalCoursesNumber } =
+    useAppSelector((state) => state.coursesManagement);
+
+  const navigateToEditCourse = (courseId: number): void => {
+    dispatch(coursesActions.getCourse({ id: courseId }));
+    navigation.navigate(AppScreenName.EDIT_COURSE);
+  };
+
+  const coursesRows = coursesWithCategory.map((item: CourseGetResponseDto) => {
+    return {
+      title: item.title,
+      category: item.category ? (
+        <CategoryCell category={item.category} />
+      ) : (
+        'Unknown'
+      ),
```

Comment:
Won't it be better to handle empty value placeholder inside `CategoryCell` component?